app/src/main/java/com/github/sdp/tarjetakuna/utils/CustomGlide.kt

```diff
@@ -0,0 +1,39 @@
+package com.github.sdp.tarjetakuna.utils
+
+import android.graphics.drawable.Drawable
+import androidx.fragment.app.Fragment
+import androidx.test.espresso.idling.CountingIdlingResource
+import com.bumptech.glide.Glide
+import com.bumptech.glide.request.target.CustomTarget
+import com.bumptech.glide.request.transition.Transition
+
+/**
+ * This class is used to load images with Glide and keep track of the loading process for testing purposes.
+ */
+object CustomGlide {
+
+    private const val RESOURCE = "GLOBAL"
+    @JvmField val countingIdlingResource = CountingIdlingResource(RESOURCE)
```

Comment:
This is only for testing right ? In this case i'd suggest to make a test class that adds this field, and override the loadDrawable in consequence, so that this CustomGlide can be used in the Application without being impacted by this IdlingRessource stuff

Reply:
Much better than my technique indeed. I will make the changes accordingly.